watch-parties/src/main/java/com/google/sps/servlets/Member.java

```diff
@@ -0,0 +1,36 @@
+package com.google.sps.servlets;
+import com.google.appengine.api.users.User;
+import java.util.Date;
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.KeyFactory;
+import com.google.appengine.api.datastore.Key;
+public class Member {
+  private Key id;
+  private String nickname;
+  private boolean partyLeader;
+  private User user;
+
+  public Member(User u, boolean leader, String name){
```

Comment:
Keep the params the same as the instance variables and use the "this" keyword to differentiate between the parameter and instance variable.

public Member(User user, boolean partyLeader, String nickname)
 
Ex. this.user = user;